pkg/controller/queuejobresources/genericresource/genericresource.go

```diff
@@ -319,29 +328,30 @@ func (gr *GenericResources) SyncQueueJob(aw *arbv1.AppWrapper, awr *arbv1.AppWra
 	}
 
 	// Get the related resources of created object
-	var thisObj *unstructured.Unstructured
+	// var thisObj *unstructured.Unstructured
 	var err1 error
-	if namespaced {
-		thisObj, err1 = dclient.Resource(rsrc).Namespace(namespace).Get(context.Background(), name, metav1.GetOptions{})
-	} else {
-		thisObj, err1 = dclient.Resource(rsrc).Get(context.Background(), name, metav1.GetOptions{})
-	}
+	// if namespaced {
+	// 	thisObj, err1 = dclient.Resource(rsrc).Namespace(namespace).Get(context.Background(), name, metav1.GetOptions{})
+	// } else {
+	// 	thisObj, err1 = dclient.Resource(rsrc).Get(context.Background(), name, metav1.GetOptions{})
+	// }
 	if err1 != nil {
```

Comment:
Why not comment these lines too? This error is no longer set by the code.

Reply:
Same for line 354. We can just comment this whole section and call `return []*v1.Pod{}, nil`.